package.json

```diff
@@ -31,34 +31,34 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "chalk": "^2.3.1",
+    "chalk": "^2.4.1",
     "cheerio": "^1.0.0-rc.2",
     "clipboardy": "^1.2.3",
-    "core-validate-commit": "^3.5.0",
-    "execa": "^0.10.0",
+    "core-validate-commit": "^3.6.3",
+    "execa": "^1.0.0",
     "figures": "^2.0.0",
-    "fs-extra": "^6.0.1",
+    "fs-extra": "^7.0.1",
     "ghauth": "^3.2.1",
-    "listr": "^0.14.1",
-    "lodash": "^4.17.10",
+    "listr": "^0.14.2",
+    "lodash": "^4.17.11",
     "mkdirp": "^0.5.1",
-    "node-fetch": "^2.1.1",
-    "ora": "^2.0.0",
+    "node-fetch": "^2.2.1",
+    "ora": "^3.0.0",
     "read": "^1.0.7",
     "rimraf": "^2.6.2",
-    "yargs": "^11.0.0"
+    "yargs": "^12.0.2"
   },
   "devDependencies": {
-    "eslint": "^4.17.0",
-    "eslint-config-standard": "^11.0.0",
-    "eslint-plugin-import": "^2.8.0",
-    "eslint-plugin-node": "^6.0.1",
-    "eslint-plugin-promise": "^3.6.0",
-    "eslint-plugin-standard": "^3.0.1",
+    "eslint": "^5.9.0",
+    "eslint-config-standard": "^12.0.0",
```

Comment:
I actually start to think we should use something else (like the one ported from node core) instead..

Reply:
I agree. Using Standard with semicolons is... unusual 😄